custom_resources/logs.py

```diff
@@ -0,0 +1,24 @@
+from six import string_types
+
+from .LambdaBackedCustomResource import LambdaBackedCustomResource
+
+
+class ResourcePolicy(LambdaBackedCustomResource):
```

Comment:
Voeg als comment/docstring nog toe waarom deze CustomResource nodig is. In dit geval is het waarschijnlijk omdat er vandaag nog geen built-in resource bestaat die dit doet?